src/consensus/aft/revealed_nonces.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+#include "kv/map.h"
+
+#include <msgpack/msgpack.hpp>
+#include <string>
+#include <vector>
+
+namespace aft
+{
+  using Nonce = std::array<uint8_t, 32>;
```

Comment:
Sha256::SIZE or something like that.

Reply:
Discussed offline we need something common for CCF. https://github.com/microsoft/CCF/issues/1690